packages/node-resolve/src/resolveImportSpecifiers.js

```diff
@@ -178,7 +25,7 @@ async function resolveId({
   preserveSymlinks,
   useBrowserOverrides,
   baseDir,
-  moduleDirectories
+  moduleDirectories = ['node_modules']
```

Comment:
When would this be undefined? Don’t we set a default in index?

Reply:
You're right it's set in the index. I removd it.